docs/source/quickstart/script.md

```diff
@@ -0,0 +1,262 @@
+|                               |                 |                 | model selection as a post step.                    |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --run_notes                   | <class 'str'>   |                 | Notes for the current training run. Use this       |
+|                               |                 |                 | argument to add short description of the model     |
+|                               |                 |                 | training run that helps in identifying the run     |
+|                               |                 |                 | later.                                             |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --models_dir                  | <class 'str'>   | models/         | Path to save the model. Will be expanded to        |
+|                               |                 |                 | models_dir/run_id                                  |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --logs_dir                    | <class 'str'>   | logs/           | Path to save the training/inference logs. Will be  |
+|                               |                 |                 | expanded to logs_dir/run_id                        |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --checkpoint_model            | <class 'bool'>  | True            | Whether to save model checkpoints at the end of    |
+|                               |                 |                 | each epoch. Recommended - set to True              |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --train_pcent_split           | <class 'float'> | 0.8             | Percentage of all data to be used for training.    |
+|                               |                 |                 | The remaining is used for validation and testing.  |
+|                               |                 |                 | Remaining data is split in half if val_pcent_split |
+|                               |                 |                 | or test_pcent_split are not specified.             |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --val_pcent_split             | <class 'float'> | -1              | Percentage of all data to be used for testing.     |
++-------------------------------+-----------------+-----------------+----------------------------------------------------+
+| --test_pcent_split            | <class 'float'> | -1              | Percentage of all data to be used for testing.     |
```

Comment:
what is the meaning of the -1 default value ?

Reply:
It is not supported currently. Nothing happens if you pass this value. This is a TODO for ml4ir.